accelerator/cli_info.h

```diff
@@ -49,30 +49,33 @@ typedef enum ta_cli_arg_value_e {
   VERBOSE,
 } ta_cli_arg_value_t;
 
-typedef enum ta_cli_arg_requirement_e { NO_ARG, REQUIRED_ARG, OPTIONAL_ARG } ta_cli_arg_requirement_t;
+enum ta_cli_arg_requirement_e { NO_ARG, REQUIRED_ARG, OPTIONAL_ARG };
```

Comment:
Since those value in `ta_cli_arg_requirement_e` will be directly assign to `long_option.hash_arg`. Shall we just use the Marcoes defined in `getopt_ext.h`?